recipe/meta.yaml

```diff
@@ -15,14 +15,11 @@ source:
 
 build:
   number: 0
```

Comment:
Thank you @mariusvniekerk for moving the conda-forge recipes to conda build 3 format. But my advice is that you do a complete move and also add account for run_exports, i.e., make sure host dependencies with run exports are not in run requirements and add run_exports for the recipe if appropriate. For example, here you are missing the run_exports. Please add them similar to: https://gitlab.idiap.ch/bob/bob.conda/blob/2fb6e571a760892e2447494cb52944011fbf302f/conda/libsvm/meta.yaml#L18

Reply:
libsvm is extremely abi compatible.  Additionally it is versioned as just x so having a max-pin of x.x doesn't really do much for us here.

https://abi-laboratory.pro/index.php?view=timeline&l=libsvm